triggerGHR/config/nhc_fault_dictionary.cfg

```diff
@@ -0,0 +1,23 @@
+NHC2001 = Resource.Hpc.Unhealthy.HpcGenericFailure #Use this to report issues that do not fall under any other HPC categories. 

```

Comment:
Is this supposed to be bash syntax? There should be no spaces between the "=", variable, and value.

i.e. NHC1234=Resource.Hpc.Unhealthy.Value

Reply:
No this is not a bash syntax, it's just like a file with key value pairs..